### PR TITLE
Upgrading version to 0.5.0.

### PR DESCRIPTION
**Pull request by dhermes**
Also changing author to something more official / encompassing. (or at least aspiring to do so)

@jgeewax What should the author be?

----

Using last release PR (#723) as a template.

### CODE REVIEW

**Comment by dhermes** [comment]
Running

```bash
$ git log -1 --pretty=%H
df3a801fa997788fe244c7d5216ba9b10db08316
$ git log 0.4.3..HEAD \
> | grep 'Merge pull request ' \
> | awk -F ' ' '{print $4}' \
> | sort
```

yields

#719
#725
#731
#732
#733
#735
#737
#740
#742
#743
#746
#747
#748
#749
#751
#756
#760
#763
#765
#766
#767
#768
#769
#770
#771
#772
#773
#774
#779
#780
#782
#783
#784
#785
#788
#789
#790
#792
#793
#794
#795
#800
#801
#802
#803
#804
#806
#807
#811

File: setup.py [hunk 1]
```diff
@@ -23,10 +23,10 @@
 
 setup(
     name='gcloud',
-    version='0.4.3',
+    version='0.5.0',
     description='API Client library for Google Cloud',
-    author='JJ Geewax',
-    author_email='jj@geewax.org',
+    author='FOO BAR',
+    author_email='foo@bar.org',
```
Comment (tseaver) [inline setup.py]: Maybe the author should read 'Google Cloud Developers', with an e-mail address which points to a mailing list?

Comment (jgeewax) [inline setup.py]: How about

```
author='Google Cloud Platform'
author_email='jjg@google.com'
```

Until I get a proper mailing list for this set up....

thoughts?

**Comment by dhermes** [comment]
This is my plan for the release notes on the tag:

```
-   Adding alpha support for `pubsub`!
-   Adding `Blob.reload()` and `Bucket.reload()` for getting / refreshing
    metadata.
-   Adding `Bucket.create()` for inserting a new bucket (equivalent to
    `storage.create_bucket()`).
-   Removing `Bucket.new_blob`.
-   Removing `Bucket.batch` and `Blob.batch` context managers (functionality
    usurped by `storage.Batch`).
-   Make `(Bucket|Blob).exists()` more efficient (using `fields`).
-   Unbundling `project` string in storage `Connection`s.
-   Adding default `project` lazy loading for storage and pubsub.
-   Swapping argument order in `Bucket` constructor to allow
    `Bucket('bucket-name')` in implicit case.
-   Fixing bug where metadata responses in `Blob.upload*` and `Bucket.copy_blob`
    were not being used.
-   Getting system tests (regression) passing in Python 3. Blocked on `httplib2`
    release.
-   Removing `properties=` from `Blob` and `Bucket` constructors.
-   Fixing [bug][1] in reported exceptions for JSON-based (aka Discovery) APIs.
-   Replacing `Bucket.(get|update)_cors` and `Bucket.(get|update)_lifecycle`
    and with properties.
-   Making `Bucket/Blob` getters/setters operator on local data only, rather
    than sending requests. This includes `Bucket.(get|enable|disable)_logging`,
    `Bucket.(configure|disable)_website`, `Bucket.lifecycle_rules` and
    `Bucket.cors`.
-   Removing `Bucket.get_default_object_acl`.
-   Adding method `gcloud.datastore.helper.find_true_dataset_id` which allows
    converting an aliased dataset ID (e.g. `foo`) to an unaliased / true ID
    (e.g. `s~foo`).
-   Combining `Bucket.get_all_blobs` and `Bucket.iterator` into
    `Bucket.list_blobs`.
-   Renaming `storage.get_all_buckets` to `storage.list_buckets`.
-   Returning native datetime objects for `Blob.time_deleted`, `Blob.updated`,
    and `Bucket.time_created`.

[1]: https://github.com/GoogleCloudPlatform/gcloud-python/pull/767
```

File: setup.py (hunk 1 above)
Comment (dhermes) [inline setup.py]: SGTM. How about `jjg+gcloud-python@google.com` so you can make a GMail filter.

**Comment by tseaver** [comment]
LGTM

**Pull request merged**